ui/src/hosts/components/SearchBar.tsx

```diff
@@ -9,6 +9,7 @@ interface Props {
   width?: number
   placeholder: string
   onSearch: (searchTerm: string) => void
+  dataTest?: string
```

Comment:
A nit: I would rather see testId herein and in all other places ... because it clearly names what the property is used for. I can live with this though :)